src/modules/categories/components/Categories.js

```diff
@@ -163,7 +163,8 @@ class Categories extends React.Component<PropsType> {
       </SpaceBetween>
     }
     // some level between, we want to display a list
-    return <SpaceBetween>
+    return (
+      <>
```

Comment:
The SpaceBetween is needed to give `View` (the actual content) and `SiteHelfulBox` (the feedback prompt) appropriate spacing. Reintroduce it here.